src/components/navigation/navigation.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import style from './style/navigation';
+
+class Navigation extends Component {
+  renderSignInOrOut() {
+    let node = (
+      <li className={ style.listItem }>
+        <Link to='/account/signin'>Sign In</Link>
```

Comment:
Should be consistent about string literals in components.
Consider adding this eslint rule: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-literals.md